sdk/solidity/package.json

```diff
@@ -4,15 +4,15 @@
   "description": "Solidity SDK for interacting with Express Relay contracts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/pyth-network/pyth-crosschain",
-    "directory": "express_relay/sdk/solidity"
+    "url": "https://github.com/pyth-network/per",
+    "directory": "sdk/solidity"
   },
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
     "format": "prettier --write .",
-    "generate-abi": "generate-abis IExpressRelay IExpressRelayFeeReceiver",
+    "generate-abi": "forge inspect IExpressRelay abi > abis/IExpressRelay.json && forge inspect IExpressRelayFeeReceiver abi > abis/IExpressRelayFeeReceiver.json",
```

Comment:
Do we need `forge` to work with this command? Do we have it in dependency of solidity sdk?

Reply:
yes, it's not a npm dependency.
I need to port the workflow which runs these commands and I need to install forge in that workflow.